src/js/actions/transform.js

```diff
@@ -231,7 +235,42 @@ define(function (require, exports) {
         // Used to calculate the new top/left positions of all layers in relation to top/left
         // of the group
         var overallWidthRatio = size.w ? size.w / overallBounds.width : 0,
-            overallHeightRatio = size.h ? size.h / overallBounds.height : 0;
+            overallHeightRatio = size.h ? size.h / overallBounds.height : 0,
+            overallLeftOffset = 0,
+            overallTopOffset = 0,
+            direction;
+
+        // This is the case if the width input was changed
+        if (size.hasOwnProperty("w")) {
+            direction = refPoint.charAt(0);
+            switch (direction) {
+                case "l":
+                    break;
```

Comment:
Is this intended for showing the complete cases? can you leave a comment to explain the reason?